src/libs/dutil/regutil.cpp

```diff
@@ -935,3 +935,59 @@ extern "C" HRESULT DAPI RegQueryKey(
 LExit:
     return hr;
 }
+
+/********************************************************************
+RegKeyReadNumber - reads a DWORD registry key value as a number from
+a specified subkey.
+
+*********************************************************************/
+extern "C" HRESULT DAPI RegKeyReadNumber(
+    __in HKEY hk,
+    __in_z LPCWSTR wzSubKey,
+    __in_z_opt LPCWSTR wzName,
```

Comment:
I think you want __in_z - assuming NULL would accesses the default value, which I don't think can be a number.